server/src/grpc_clients.rs

```diff
@@ -0,0 +1,40 @@
+use svc_pricing_client::pricing_grpc::pricing_client::PricingClient;
+use svc_scheduler_client::svc_scheduler::scheduler_client::SchedulerClient;
+use svc_storage_client_grpc::client::storage_rpc_client::StorageRpcClient;
+use tonic::transport::Channel;
+
+pub use svc_storage_client_grpc::client::VertiportFilter;
+
+#[derive(Clone, Debug)]
+pub struct GrpcClients {
+    pub scheduler: SchedulerClient<Channel>,
+    pub storage: StorageRpcClient<Channel>,
+    pub pricing: PricingClient<Channel>,
+}
```

Comment:
So the problem with GRPC clients are: they need to be mutable. And if there is a chance that requests might be called concurrently, clients need to be cloned. See this discussion: https://github.com/hyperium/tonic/issues/285 and TLDR.

This was my solution in svc-scheduler to call svc-storage-client - to have global Client and clone it before each request.
https://github.com/Arrow-air/svc-scheduler/compare/roman/dw-212/svc-scheduler-v1-2#diff-01fd5002f7585d6f2b853a8c83e6c0a1acb2b16b6e6f8f581fb3365377d331bcR31-R41

Reply:
Now cloning clients, mutexed connection block